beacon-chain/node/node.go

```diff
@@ -251,6 +243,16 @@ func New(cliCtx *cli.Context, cancel context.CancelFunc, opts ...Option) (*Beaco
 	beacon.verifyInitWaiter = verification.NewInitializerWaiter(
 		beacon.clockWaiter, forkchoice.NewROForkChoice(beacon.forkChoicer), beacon.stateGen)
 
+	pa := peers.NewAssigner(beacon.fetchP2P().Peers(), beacon.forkChoicer)
```

Comment:
What was the reason to change the ordering here ?

Reply:
Because backfill now depends on initwaiter (ie blob validation), which depends on stategen.